gemfiles/rails_6_1.gemfile

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+eval_gemfile "../Gemfile"
+
+if Gem::Version.new(RUBY_VERSION.dup) >= Gem::Version.new("3.1")
```

Comment:
Can you explain why this is necessary?

Reply:
The `mail` gem has an issue discussing the need here - https://github.com/mikel/mail/pull/1439 .  And the Rails team discusses it here - https://github.com/rails/rails/pull/44600 .

These gems are not included by default in Ruby 3.1, so they need to be explicitly included.  Attempting to explicitly include them in older Rubies causes issues, so the invoking Gemfile is supposed to explicitly include them.